api/VRDisplay.json

```diff
@@ -0,0 +1,711 @@
+                  "version_added": true,
+                  "flag": {
+                    "type": "preference",
+                    "name": "WebVR"
+                  },
+                  "notes": "Only works on desktop in an <a href=\"https://webvr.info/get-chrome/\">experimental version of Chrome</a> (other builds won't return any devices when <code>Navigator.getVRDisplays()</code> is invoked)."
+                },
+                "edge": {
+                  "version_added": true
+                },
+                "firefox": {
+                  "version_added": "55",
+                  "notes": "Currently only Windows support is enabled by default. Mac support is <a href=\"https://hacks.mozilla.org/2017/06/announcing-webvr-on-mac/\">available in Firefox Nightly</a>."
+                },
+                "ie": {
+                  "version_added": false
+                },
+                "webview_android": {
+                  "version_added": false
+                },
+                "edge_mobile": {
+                  "version_added": null
+                },
+                "ie_mobile": {
+                  "version_added": false
+                },
+                "chrome_android": {
+                  "version_added": true,
+                  "notes": "Currently supported only by Google Daydream."
+                }
+              }
+            }
+          }
+        },
+        "submitFrame": {
+          "__compat": {
+            "basic_support": {
+              "support": {
+                "chrome": {
+                  "version_added": true,
+                  "flag": {
+                    "type": "preference",
+                    "name": "WebVR"
+                  },
+                  "notes": "Only works on desktop in an <a href=\"https://webvr.info/get-chrome/\">experimental version of Chrome</a> (other builds won't return any devices when <code>Navigator.getVRDisplays()</code> is invoked)."
+                },
+                "edge": {
+                  "version_added": true
+                },
+                "firefox": {
+                  "version_added": "55",
+                  "notes": "Currently only Windows support is enabled by default. Mac support is <a href=\"https://hacks.mozilla.org/2017/06/announcing-webvr-on-mac/\">available in Firefox Nightly</a>."
+                },
+                "ie": {
+                  "version_added": false
+                },
+                "webview_android": {
+                  "version_added": false
+                },
+                "edge_mobile": {
+                  "version_added": null
+                },
+                "ie_mobile": {
+                  "version_added": false
+                },
+                "chrome_android": {
+                  "version_added": true,
+                  "notes": "Currently supported only by Google Daydream."
+                }
+              }
+            }
+          }
+        },
+        "getPose": {
```

Comment:
Three things in this file:

- No feature has a status object here, but as `getPose` is deprecated I think it needs a [status object](https://github.com/mdn/browser-compat-data/blob/master/compat-data-schema.md#status-information) at least.
```
"status": {
  "experimental": true,
  "standard_track": true, (or false?)
  "deprecated": true
}
```
- Also, order the features alphabetically, `getPose` is at the end at the moment.

- Why aren't `hardwareUnitId` and `getImmediatePose` in the data (with a status object)? Are they removed from the spec and browsers? Why are they on MDN in that case?